src/common/utils.h

```diff
@@ -29,6 +29,8 @@
 #include "../common/cbasetypes.h"
 #include "../common/mmo.h"
 
+constexpr size_t GetPlayerNameLength() { return 15; }
```

Comment:
i'm not sure if it should explicitly say "player" - technically, the size is also 15 for every entity in the packets isn't it?  up for discussion, that one

but either way this should just be a variable instead of a function `constexpr size_t max_name_length = 15;`

Reply:
mobs and npc's can have longer names, unless we shove phone text through player chat packets

Reply:
For packet purposes, they're all limited to 15 characters + null terminator, anything more gets truncated.

Reply:
sure, but not `Get` because it won't be a function

Reply:
Do you want me to prefix it with anything to signify that it's a constant?

Reply:
Also, please correct me if I'm wrong, but packets are cleared out with 0 before being constructed, yes?

Reply:
packets are zero initialised, yep
I don't think it needs a prefix, the type is constexpr and that's probably good enough

Reply:
ez lyfe. I'm not going to attempt to limit any packets that are using EXPLICITLY a CCharEntity name because it's already limiting the length to 15 characters. Anything else, e.g. BaseEntity, packetname, etc, I will make sure it limits the amount of data being copied into the packet if it was doing so before.

Auction house is a weird exception. Not sure why it was 16, but it uses CCharEntity so letting the limit in SetName handle it.